assets/persistence/zone_layouts.gd

```diff
@@ -0,0 +1,14 @@
+extends PersistentDataFile
+
+func get_file_name() -> String:
+	return "res://assets/content/initial_zones.json"
```

Comment:
After further consideration, this will not work. Most operating systems will take issue with modifying files in the game. We should instead add a way to initialise from a game file before defaulting to the save file.

Reply:
These files dont ever get modified